chichiwang/exercise-js
# Add Animated logo and messaging to pages: Error, NotFound

## Description
Linked to Issue: #109 
Update `pages/ErrorPage` and `pages/NotFound` to add the animated logo and proper messaging and styling.

## Changes
* Update `pages/ErrorPage/index.tsx`
  * Wrap content in a `ContentBlock`
  * Display the `AnimatedLogo`
  * Add messaging about an error occurring
* Update `pages/NotFound/index.tsx`
  * Wrap content in a `ContentBlock`
  * Display the `AnimatedLogo`
  * Add messaging 

## Steps to QA
* Pull down and switch to this branch
* Run the app and verify in browser
  * The new UI on the `/error` page looks good
  * The new UI on the NotFound page looks good (use any non-registered routes, ex: `/foobar`)
